packages/kit/src/views/DAppConnection/hooks/useDappAccountSwitch.ts

```
@@ -241,74 +245,71 @@ export function useDappAccountSwitch({
+        isAvailableAccount = false;
+      }
 
-        if (!isSameAccount && !isSyncing && isAvailableAccount) {
-          const scopes = getScopeFromImpl({
-            impl: result.connectedAccountsInfo[0].networkImpl,
+      if (!isSameAccount && !isSyncing && isAvailableAccount) {
+        const scopes = getScopeFromImpl({
+          impl: result.connectedAccountsInfo[0].networkImpl,
+        });
+        // update connected accounts info from home
+        if (scopes && scopes.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)